### ISSUE TITLE
Resolve Fancybox accessibility issues

### ISSUE THREAD

**Issue by ptagell** [unimelb/The-University-of-Melbourne-Web-Templates#22]
Currently fancybox is not accessible due to keyboard traps (more details to follow), so this needs to be resolved in order to meet WCAG 2.0 AA standards.

**Comment by tobyhede** [comment]
From memory, Andrew Normand's recommendation was to use a lightbox plugin built with accessibility in mind. Will trawl out the link - there is sort of "standard" one.
